cmd/server/assets/realm.html

```diff
@@ -30,7 +31,30 @@ <h1>Realm settings</h1>
           <div class="form-group row">
             <label for="name" class="col-sm-3">Name:</label>
             <div class="col-sm-9">
-              <input type="text" id="name" name="name" class="form-control" value="{{.realm.Name}}" required />
+              <input type="text" id="name" name="name" class="form-control{{if $realm.ErrorsFor "name"}} is-invalid{{end}}" value="{{$realm.Name}}" />
+              {{if $realm.ErrorsFor "name"}}
+              <div class="invalid-feedback">
+                {{joinStrings ($realm.ErrorsFor "name") ", "}}
+              </div>
+              {{end}}
+            </div>
+          </div>
+
+          <div class="form-group row">
+            <label class="col-sm-3">Allowed reports:</label>
```

Comment:
We use "report type" and "test type" interchangeable in most places. We should probably standardize.